colcon_in_container/verb/in_container.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2023 Canonical, Ltd.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from abc import ABC
+
+from colcon_core.plugin_system import satisfies_version
+from colcon_core.verb import VerbExtensionPoint
+
+
+class InContainer(ABC, VerbExtensionPoint):
```

Comment:
Not necessary, but in this case it also abstract the `VerbExtensionPoint`. In the future we could also put in this class the generic `add_arguments` but I doubt it would be more.